### PR TITLE
docs: Update uno solution app structure

### PR DESCRIPTION
**Pull request by jeromelaban**

### CODE REVIEW

File: doc/articles/uno-app-solution-structure.md [hunk 1]
```diff
@@ -12,13 +12,48 @@ After creating a new solution called `MyApp`, it will contain the following proj
...
 
-This project contains all the code for your application, alongside platform-specific code found in the `Platforms` folder. Each of these folders is only processed as part of their target environment.
+You'll find below details about the contents of the solution.
+
+### Platforms
+
+The Platforms folder contains platform specific files for targets supported by Uno Platform:
+
+- `Desktop` is using the [Uno Platform Skia Desktop](xref:Uno.Features.Uno.Sdk) support for Windows 7+, Linux and macOS
+- `Android` contains files and assets specific to the Android Phones, Tablets, TVs and watches
```
Comment (agneszitte) [inline doc/articles/uno-app-solution-structure.md:24]:
```suggestion
- `Desktop` uses the [Uno Platform Skia Desktop](xref:Uno.Features.Uno.Sdk) support for Windows 7+, Linux, and macOS
- `Android` contains files and assets specific to Android Phones, Tablets, TVs, and watches
```

File: doc/articles/uno-app-solution-structure.md [hunk 2]
```diff
@@ -12,13 +12,48 @@ After creating a new solution called `MyApp`, it will contain the following proj
...
+- `iOS` targets Apple's iOS devices, Phones and Tablets
+- `MacCatalyst` targets the macOS Catalyst platform
+- `WebAssembly` targets the browser using WebAssembly
+- `Windows` targets the [Windows App SDK](https://developer.microsoft.com/en-us/windows/downloads/windows-sdk/) to run on Windows
+
+### Properties
+
+This folder contains the debug profile configuration. This is used to choose where to debug your Skia Desktop app (Local Windows or WSL), the Web Server for your WebAssembly app or Package and Unpackaged modes for Windows App SDK.
```
Comment (agneszitte) [inline doc/articles/uno-app-solution-structure.md:32]:
```suggestion
This folder contains the debug profile configuration. This is used to choose where to debug your Skia Desktop app (Local Windows or WSL), the Web Server for your WebAssembly app, or Package and Unpackaged modes for Windows App SDK.
```

File: doc/articles/uno-app-solution-structure.md [hunk 3]
```diff
@@ -12,13 +12,48 @@ After creating a new solution called `MyApp`, it will contain the following proj
...
+
+### Assets
+
+This folder contains [all the assets](xref:Uno.Features.Assets) (images, splash screens, data, ...) that are published as part of the app.
+
+## Other project files
+
+- `App.xaml` and `App.xaml.cs` are the common entry point for the app. The generic app setup runtime code generally goes here.
+- `MainPage.xaml` and `MainPage.xaml.cs` contains the main UI of the app.
```
Comment (agneszitte) [inline doc/articles/uno-app-solution-structure.md:41]:
```suggestion
- `MainPage.xaml` and `MainPage.xaml.cs` contain the main UI of the app.
```

File: doc/articles/uno-app-solution-structure.md [hunk 4]
```diff
@@ -12,13 +12,48 @@ After creating a new solution called `MyApp`, it will contain the following proj
...
+- `GlobalUsings.cs` contains the [global usings](https://learn.microsoft.com/en-us/dotnet/csharp/language-reference/keywords/using-directive#global-modifier) define for the project.
+- `app.manifest` contains [Windows specific configuration](https://learn.microsoft.com/en-us/windows/win32/sbscs/application-manifests) for `net8.0-desktop` and `net8.0-windows` targets.
+- `Package.appxmanifest` contains metadata about the application such as the app name and description.
+
+### Solution items
+
+This solution folder contains configuration for the whole solution:
```
Comment (agneszitte) [inline doc/articles/uno-app-solution-structure.md:48]:
```suggestion
This solution folder contains the configuration for the whole solution:
```

File: doc/articles/uno-app-solution-structure.md [hunk 5]
```diff
@@ -12,13 +12,48 @@ After creating a new solution called `MyApp`, it will contain the following proj
...
+
+- `global.json` contains the [.NET SDK configuration](https://learn.microsoft.com/en-us/dotnet/core/tools/global-json) as well as the Uno.Sdk version to use. See [our documentation](xref:Uno.Features.Uno.Sdk) on how to update it.
+- `Directory.Build.props` and `Directory.Build.targets` contain common solution configuration applied to all projects.
```
Comment (agneszitte) [inline doc/articles/uno-app-solution-structure.md:51]:
```suggestion
- `Directory.Build.props` and `Directory.Build.targets` contain common solution configurations applied to all projects.
```

**Comment by jeromelaban** [comment]
@mergifyio backport release/stable/5.2

**Pull request merged**